src/componentize.js

```diff
@@ -193,7 +193,7 @@ export async function componentize(jsSource, witWorld, opts) {
           'weval',
           `--cache-ro ${aotCache}`,
           '-w',
-          `-i ${engine}`,
+          `-i ${input}`,
           `-o ${output}`,
           `--dir ${dirname(input)}`
```

Comment:
I believe all of the arguments to Weval should be exactly the same as the arguments to Wizer, as this was how Weval was designed. So I'd actually make sure all the flags match with the Wizer lines below here.